From b8b2df2f412a1473e449abeb46142f11cf8850cc Mon Sep 17 00:00:00 2001
From: dkl <dkl@rock-chips.com>
Date: Sat, 13 Dec 2014 12:27:04 +0800
Subject: [PATCH] rk3368: dts: init gpll rate before clk_core_b and clk_core_l

It's better to init gpll rate before clk_core_b and clk_core_l,
because gpll will be used when clk_core_b and clk_core_l change rate.

Signed-off-by: dkl <dkl@rock-chips.com>
---
 arch/arm64/boot/dts/rk3368.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rk3368.dtsi b/arch/arm64/boot/dts/rk3368.dtsi
index de0fa254c701..1afc30dbf3ac 100644
--- a/arch/arm64/boot/dts/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rk3368.dtsi
@@ -450,8 +450,8 @@
 			<&clk_uart_pll &clk_gpll>, <&aclk_gpu &clk_cpll>,
 			<&clk_cs &clk_gpll>;
 		rockchip,clocks-init-rate =
-			<&clk_core_b 792000000>,	<&clk_core_l 600000000>,
-			<&clk_gpll 576000000>,		<&clk_cpll 400000000>,
+			<&clk_gpll 576000000>,		<&clk_core_b 792000000>,
+			<&clk_core_l 600000000>,	<&clk_cpll 400000000>,
 			/*<&clk_npll 500000000>,*/	<&aclk_bus 300000000>,
 			<&hclk_bus 150000000>,		<&pclk_bus 75000000>,
 			<&clk_crypto 150000000>,	<&aclk_peri 300000000>,
-- 
2.35.3

